static/noto.css

```diff
@@ -0,0 +1,115 @@
+/* Arabic */
+@font-face {
+    font-family: "Noto Serif";
+    font-weight: 400;
+    src: url("fonts/arabic/NotoNaskhArabic-Regular.woff2");
+    unicode-range: U+0600-06FF;
+}
+
+@font-face {
+    font-family: "Noto Sans";
+    font-weight: 400;
+    src: url("fonts/arabic/NotoSansArabic-Regular.woff2");
+    unicode-range: U+0600-06FF;
+}
+
+@font-face {
+    font-family: "Noto Sans";
+    font-weight: 700;
+    src: url("fonts/arabic/NotoSansArabic-Bold.woff2");
+    unicode-range: U+0600-06FF;
+}
+
+@font-face {
+    font-family: "Noto Sans";
+    font-weight: 900;
+    src: url("fonts/arabic/NotoSansArabic-Black.woff2");
+    unicode-range: U+0600-06FF;
+}
+
+/* Armenian */
```

Comment:
Added this font to fix this later in Webapp: https://khanacademy.atlassian.net/browse/IC-153